components/layout_thread_2020/lib.rs

```diff
@@ -1082,7 +1082,8 @@ impl LayoutThread {
             driver::traverse_dom(&traversal, token, Some(rayon_pool));
 
             let root_node = document.root_element().unwrap().as_node();
-            let box_tree = rayon_pool.install(|| BoxTreeRoot::construct(traversal.context(), root_node));
+            let box_tree =
+                rayon_pool.install(|| BoxTreeRoot::construct(traversal.context(), root_node));
```

Comment:
Why did that change? Oh, forgot to run fmt on a previous commit?

Reply:
Probably?